plugin/trino-mariadb/src/test/java/io/trino/plugin/mariadb/TestMariaDBTableStatisticsTest.java

```diff
@@ -0,0 +1,606 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.mariadb;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.Session;
+import io.trino.SystemSessionProperties;
+import io.trino.plugin.jdbc.BaseJdbcTableStatisticsTest;
+import io.trino.sql.planner.assertions.PlanMatchPattern;
+import io.trino.sql.planner.plan.FilterNode;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.sql.TestTable;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.core.Jdbi;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static io.trino.plugin.mariadb.MariaDbQueryRunner.createMariaDbQueryRunner;
+import static io.trino.sql.planner.assertions.PlanMatchPattern.anyTree;
+import static io.trino.sql.planner.assertions.PlanMatchPattern.tableScan;
+import static io.trino.testing.sql.TestTable.fromColumns;
+import static io.trino.testing.sql.TestTable.randomTableSuffix;
+import static io.trino.tpch.TpchTable.CUSTOMER;
+import static io.trino.tpch.TpchTable.NATION;
+import static io.trino.tpch.TpchTable.ORDERS;
+import static io.trino.tpch.TpchTable.REGION;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestMariaDBTableStatisticsTest
+        extends BaseJdbcTableStatisticsTest
+{
+    protected TestingMariaDbServer mariaDbServer;
+
+    @Override
+    protected QueryRunner createQueryRunner() throws Exception
+    {
+        // From MariaDB 10.8, JSON_HB, JSON-format histograms, are accepted, so used 10.8.3 version
+        mariaDbServer = closeAfterClass(new TestingMariaDbServer("10.8.3", "JSON_HB"));
+
+        return createMariaDbQueryRunner(mariaDbServer,
```

Comment:
nit: Move `mariaDbServer` to a new line.

Reply:
Done